pkg/otlp/model/go.mod

```diff
@@ -20,17 +20,21 @@ require (
 	github.com/dustin/go-humanize v1.0.0 // indirect
```

Comment:
We can revert the changes on the `go.mod` and `go.sum` files from both Go modules, they are now not related to this patch and we will avoid introducing unrelated changes on this PR